src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/srm/RegexOptions.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 using System.Diagnostics;
 
-namespace Microsoft.SRM
+namespace System.Text.RegularExpressions.SRM
 {
     [Serializable]
     internal struct RegexOptions
```

Comment:
Now that you're merging the code, seems the SRM versions of IgnoreCase, etc can be replaced by simply using the existing enum? And maybe retaining some of your helper methods here but no more need for any "mapping" from one to the other.

Reply:
Right, removed SRM.RegexOptions.